index.html

```diff
@@ -207,6 +212,35 @@ <h3>Specific Verifiable Credentials</h3>
 that should work for most use cases.
         </p>
       </section>
+      <section>
+        <h3>Content Integrity</h3>
+        <p>
+Protecting the integrity of content is an important component of verification.
+<a>Verifiers</a> need to have confidence that the content they rely on to
+verify <a>credentials</a> doesn't change without their knowledge. This content
+may include data schemas, identifiers, public keys, etc.
+        </p>
+        <p>
+There are a number of ways to provide content integrity protection. A few of
+these are described in greater detail below.
+        </p>
+        <section>
+          <h4>Hashlinks</h4>
+
+          <p>
+Hashlink URLs can be used to provide content integrity for links to external
```

Comment:
We should add a pointer to https://tools.ietf.org/html/draft-sporny-hashlink-00 or some other resource that describes in detail what hashlinks are.

Reply:
I will add the link, and agree that the hashlink section needs more detail. I don't feel qualified to add that detail, so I will raise a separate issue for that.

Reply:
I've added a comment to Issue #31 to track this